frontend/src/components/pages/AdminDashboard/ViewCheckInPage.tsx

```diff
@@ -1,12 +1,12 @@
 import { Container } from "@chakra-ui/react";
 import React from "react";
 
-import ViewDonations from "./ViewDonations";
+import ViewCheckins from "./ViewDonationsAndCheckins";
```

Comment:
This is interesting, I always thought that the name we use to export in `ViewDonationsAndCheckins`, we have to import it with the same name, but it looks like thats not the case?

Reply:
Or am i misunderstanding smth?

Reply:
I think since we're doing `export default ViewDonationsAndCheckins`, whenever you import anything (regardless of name), it will export `ViewDonationsAndCheckins`